src/Commands/check.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+/**
+ * adonis-cli
+ *
+ * (c) Harminder Virk <virk@adonisjs.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+*/
+
+const colors = require('colors')
+const execSync = require('child_process').execSync
+const semver = require('semver')
+
+const requiredNodeVersion = '>=4.0.0'
+const requiredNpmVersion = '>=3.0.0'
+
+module.exports = function (argv) {
+  const nodejsVersion = process.version.replace('v', '')
+  const npmVersion = execSync('npm -v', {input: 'string', 'encoding': 'utf-8'})
+
+  console.log('-----------AdonisJS requirements check--------------')
+  console.log(`${colors.green('AdonisJS needs Node.JS >= 4.0.0 and npm >= 3.0.0\n')}`)
```

Comment:
Can you just use the values from `requiredNodeVersion` and `requiredNpmVersion`, instead of using hard coded versions?